### PR TITLE
Add activities table and activity list to user profile

### PR DESCRIPTION
**Pull request by kian1213**
Hi sir, please review my pull request. Thanks po

### CODE REVIEW

**Comment by codegracie** [comment]
Ok

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -13,4 +13,8 @@ def link_to_add_fields name, f, association, cssClass, title
     link_to name, "#", onclick: h("add_fields(this, \"#{association}\",
       \"#{escape_javascript(fields)}\")"), class: cssClass, title: title, remote: true
   end
+
+  def datetime date
+    date.strftime("%Y/%d/%m")
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: store date format in i18n, remove `()`

File: app/models/activity.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Activity < ActiveRecord::Base
+  belongs_to :user
```
Comment (dieunb) [inline app/models/activity.rb]: imagine, i have 100 model need log activity, and how do you do?

Comment (kian1213) [inline app/models/activity.rb]: Maybe I will use polymorphic association? Should I implement polymorphic here? I just assume, User table and Lesson table will use activity log. Thanks po

**Pull request merged**